### DISCUSSION TITLE
Move preview_text method to XorgProfile instead of subclasses implementing it

### DISCUSSION THREAD

**Discussion by lavafroth** [archlinux/archinstall#1912] [Ideas]
This is a rather simple suggestion relating to the desktop profiles in `archinstall/default_profiles/desktops`.

We might move `preview_text` method to `XorgProfile` class in `archinstall/default_profiles/xorg.py` instead of subclasses implementing the same method. This can dry up some code as new desktop profiles will most probably want to stick to a bunch of packages due for installation.

Of course, if the desktop profiles choose to, they may always override this behavior by implementing the method.

**Comment by svartkanin** [comment]
I think that's a good change, feel free to raise a PR
